addons/common/functions/fnc_removeCanInteractWithCondition.sqf

```diff
@@ -17,7 +17,7 @@
 
 params ["_conditionName"];
 
-_conditionName = toLower _conditionName;
+_conditionName = toLowerANSI _conditionName;
```

Comment:
Same here.

Reply:
See comment on find, applicable to all of these.